gym_xiangqi/examples/game_mode.py

```diff
@@ -26,6 +26,7 @@ def main():
             player = "You"
             piece, start, end = env.user_move_info
             piece = PIECE_ID_TO_NAME[piece]
+            end = env.game.end_pos
```

Comment:
I don't know where this came from....???? I didn't even touch this file and here it is a new line of code out of nowhere. Do we need this line?

Reply:
Nope we don't need this line. Basically, I removed it in my PR, but in your branch it's still there. It's kinda weird that it's doing this though, did you rebase or merge and resolved some conflict?

Reply:
No, i cloned it from main...creepy